.npmignore

```diff
@@ -1 +1,9 @@
-/src/
+src
+test
+.editorconfig
+.eslintrc
+.nvmrc
+.travis.yml
+api.md
+changelog.md
+license
```

Comment:
Changelog and license files are always included by NPM, so those two lines should be removed:

> Certain files are always included, regardless of settings:
> 
> - package.json
> - README (and its variants)
> - CHANGELOG (and its variants)
> - LICENSE / LICENCE